### PR TITLE
Updated example NLog.config to include logging of hosting lifetime messages

### LINKED ISSUE #615: Extend example to include support for Lifetime Startup Messages
We have already recognized it is important to perform logging of **Lifetime Startup Messages**:

https://github.com/NLog/NLog.Web/wiki/Hosting-Lifetime-Startup-Messages

But the [example code](https://github.com/NLog/NLog/wiki/Getting-started-with-ASP.NET-Core-3) is this:

```
      .ConfigureLogging(logging =>
      {
          logging.ClearProviders();   // Removes the default console-logger
          logging.SetMinimumLevel(Microsoft.Extensions.Logging.LogLevel.Trace);
      })
```

And the Nlog.config contains these rules:

```
  <rules>
    <!--All logs, including from Microsoft-->
    <logger name="*" minlevel="Trace" writeTo="allfile" />

    <!--Skip non-critical Microsoft logs and so log only own logs-->
    <logger name="Microsoft.*" maxlevel="Info" final="true" /> <!-- BlackHole without writeTo -->
    <logger name="*" minlevel="Trace" writeTo="ownFile-web" />
  </rules>
```

Maybe ConsoleTarget with Microsoft-Logger should be added to NLog.config in the tutorial, so **Lifetime Startup Messages** will work out of the box with NLog.

### PR DESCRIPTION
**Pull request by snakefoot**
Resolve #615

### CODE REVIEW

File: examples/ASP.NET Core 3/ASP.NET Core 3 - VS2019/NLog.config [hunk 1]
```diff
@@ -13,20 +13,26 @@
 
   <!-- the targets to write to -->
   <targets>
-    <!-- write logs to file  -->
+    <!-- File Target with simple details for all log messages  -->
     <target xsi:type="File" name="allfile" fileName="c:\temp\nlog-AspNetCore3-all-${shortdate}.log"
             layout="${longdate}|${event-properties:item=EventId_Id:whenEmpty=0}|${uppercase:${level}}|${logger}|${message} ${exception:format=tostring}" />
 
-    <!-- another file log, only own logs. Uses some ASP.NET core renderers -->
+    <!-- File Target with more details using some ASP.NET core renderers -->
```
Comment (304NotModified) [inline examples/ASP.NET Core 3/ASP.NET Core 3 - VS2019/NLog.config:20]: While it's of course in the rules, but not sure if starters will get that, should we not mention that only the "own logs" are logged here?

Comment (snakefoot) [reply examples/ASP.NET Core 3/ASP.NET Core 3 - VS2019/NLog.config:20]: You are welcome to improve with more details, without it turning into a novel :)

Comment (snakefoot) [reply examples/ASP.NET Core 3/ASP.NET Core 3 - VS2019/NLog.config]: Have updated the comment with "own logs"

**Comment by 304NotModified** [comment]
Thanks!

**Pull request merged**